docs/sdk/php/usage.md

```diff
@@ -0,0 +1,238 @@
+---
+id: usage
+title: Como iniciar
+tags:
+  - api
+  - php
+---
+
+## Instalando
+
+É necessário ter o PHP com versão superior à 7.4 instalado com o [Composer](https://getcomposer.org).
+
+A maneira mais fácil de instalar o SDK é com o comando abaixo que instala as dependências necessárias com o Composer:
+
+```bash
+$ composer require open-pix/php-sdk guzzlehttp/guzzle guzzlehttp/psr7
+```
+
+Dessa forma, o SDK, um cliente HTTP (`guzzlehttp/guzzle`) e uma implementação da [PSR-17](https://www.php-fig.org/psr/psr-17/) (`guzzlehttp/psr7`) serão instalados.
```

Comment:
what is the another way?

Reply:
installing another http client and another message factory. I will rephrase the sentence.